quantity.hpp

```diff
@@ -124,75 +124,76 @@ class implicit_quantity_caster : protected quantity<U,S,V,T> {
  *
  * This specialization assures quantities are created with tags.
  *
- * @tparam       Unit   the quantity's unit type
+ * @tparam  Dimension   the quantity's dimension type
  * @tparam      Scale   the quantity's scale
  * @tparam  ValueType   the quantity's value type
  * @tparam        Tag   the quantity's tag
  * @tparam      TagID   the quantity's tag ID
  * @tparam     TagNum   the quantity's tag exponents numerator
  * @tparam     TagDen   the quantity's tag exponents denominator
  */
-template< std::intmax_t            TimeNum, std::intmax_t            TimeDen
-        , std::intmax_t            MassNum, std::intmax_t            MassDen
-        , std::intmax_t          LengthNum, std::intmax_t          LengthDen
-        , std::intmax_t         CurrentNum, std::intmax_t         CurrentDen
-        , std::intmax_t      LuminosityNum, std::intmax_t      LuminosityDen
-        , std::intmax_t     TemperatureNum, std::intmax_t     TemperatureDen
-        , std::intmax_t SubstanceAmountNum, std::intmax_t SubstanceAmountDen
-        , std::intmax_t           ScaleNum, std::intmax_t           ScaleDen
-        , std::intmax_t             TagNum, std::intmax_t             TagDen
+template< std::intmax_t              TimeNum, std::intmax_t              TimeDen
+        , std::intmax_t              MassNum, std::intmax_t              MassDen
+        , std::intmax_t            LengthNum, std::intmax_t            LengthDen
+        , std::intmax_t           CurrentNum, std::intmax_t           CurrentDen
+        , std::intmax_t       TemperatureNum, std::intmax_t       TemperatureDen
+        , std::intmax_t   SubstanceAmountNum, std::intmax_t   SubstanceAmountDen
+        , std::intmax_t             ScaleNum, std::intmax_t             ScaleDen
+        , std::intmax_t               TagNum, std::intmax_t               TagDen
+        , std::intmax_t LuminousIntensityNum, std::intmax_t LuminousIntensityDen
```

Comment:
even though luminous intensity is the black sheep of the SI base units, I would not put it at the end of the template argument list ;)